src/Application/Actions/Donations/Update.php

```diff
@@ -274,11 +274,9 @@ private function addData(Donation $donation, HttpModels\Donation $donationData,
         $donation->setChampionComms($donationData->optInChampionEmail);
         $donation->setDonorBillingAddress($donationData->billingPostalAddress);
 
-        $this->donationRepository->deriveFees(
-            $donation,
-            $donationData->cardBrand,
-            $donationData->cardCountry
-        );
+        // currently this can't change the fee from what it was when donation entity was created, but
+        // we leave it here for consistency with the Create action.
```

Comment:
```suggestion
        // we call deriveFees here for consistency with the Create action, in case the derivation logic changes to depend on something we do mutate in the donation.
```